backstop/scenarios/tile.js

```diff
@@ -21,12 +21,12 @@ module.exports = [
   // Hover states
   //// Tile
   scenario('State hover', {
-    actions: [hover('#test-tile-1')],
-    selectors: ['#test-tile-1'],
+    actions: [hover('#test-tile-4')],
+    selectors: ['#demo-default'],
   }),
   scenario('State hover with thumbnail', {
```

Comment:
This might be redundant if we remove thumbnail images from tests with [M1 docker PR](https://github.com/iTwin/iTwinUI/pull/571#issuecomment-1098450285)

Reply:
Although, we should have tests for thumbnail scale change. Somehow.